lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+    JoinSession: 'Únete a la session',
+    keymissing: 'Su software(LMS/CMS)\'s falta la Key',
+    Line: 'Línea',
+    lowBandWidthSpeed: 'Su ancho de banda es demasiado bajo, estamos deteniendo la reproducción de video. Aún podrá escuchar el audio y ver la pantalla.',
+    maxCommonChat: 'Mostrar ventana de chat',
+    Max_rooms: 'Error: Límite máximo de salas alcanzado',
+    Max_users: 'Error: Límite máximo de usuarios alcanzado',
+    media: 'Media',
+    mediumBandWidthSpeed: 'Su ancho de banda es limitado, estamos reduciendo la calidad de video en función de su ancho de banda disponible.',
+    minCommonChat: 'Ocultar ventana de chat',
+    minoption: 'Ingrese al menos dos opciones',
+    microphoneNotConnected: 'El micrófono no está disponible o no está conectado correctamente',
+    // 'mictesting' : 'Si la barra de audio fluctúa mientras habla, el micrófono está funcionando.',
+    mictesting: 'El micrófono funciona, si la barra de audio anterior fluctúa <br />'
+    + 'y puedes escuchar tu propia voz mientras hablas',
+    minute: 'Minuto',
+    ModeclosingPoll: 'Modo de cerrar la encuesta :',
+    msgForDownloadStart: 'No se pueden guardar los datos. <br /> Preparando datos para descargar',
+    msgForReload: 'Vuelva a cargar esta página para continuar editando.',
+    msgForWhiteboard: 'Pizarra vacía.',
+    msgStudentForReload: 'Por favor, vuelva a cargar esta página.',
+    Multiple_login: 'Error: Acceso denegado, múltiples inicios de sesión',
+    muteAll: 'Silenciar todo',
+    muteAllAudio: 'Silenciar todo el audio',
+    mybsharedoc: 'El documento se compartirá en breve.',
+    NAME: 'NOMBRE',
+    Next: 'Siguiente',
+    note: 'Añadir nota',
+    notePlaceholder: 'Escribe tu nota aquí',
+    page: 'Página',
+    of: 'de',
+    typeQuestion: 'Escriba pregunta',
+    typeOption: 'Opción de tipo',
+    dropVideosHere : 'Suelta videos aquí',
+    nomdlroot: 'No hay URL para Moodle.',
+    normalView: 'Vista normal',
+    noResultStd: 'No tiene permiso para ver el resultado.',
+    NotAllowedError: 'Webcam desactivada',
+    NotFoundError: 'La Webcam no está disponible o no está conectada correctamente.',
+    NotReadableError: 'Asegúrese de que no se esté utilizando la misma webcam <br /> en varios navegadores o varias aplicaciones.',
+    notsupportbrowser: '{virtualclass1} no es totalmente compatible. Para la mejor experiencia, use Google Chrome.',
+    NotSupportedError: 'Solo se permiten orígenes seguros (https) para audio y video',
+    notSupportIphone: "Lo siento. La clase virtual no es compatible con teléfonos móviles.",
+    noPoll: 'No hay preguntas disponibles para la encuesta. Vaya a "Agregar nuevo" para crear una pregunta.',
+    noPollNoAdmin: 'No hay preguntas disponibles para la encuesta del sitio. Solo el administrador puede crear una encuesta a nivel del sitio.',
+    noQuiz: 'No hay cuestionario disponible. Agregue un cuestionario en su Moodle. Congrea solo admite preguntas de opción múltiple.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 134. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)